### ISSUE TITLE
Audit component styles impacted by Chrome's new form element & focus style updates

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#3131]
https://blog.chromium.org/2020/03/updates-to-form-controls-and-focus.html

Chrome made a significant update to global focus outline styles and some form element presentation. Let's look into what changed exactly and do a quick audit of the components to serve as a starting point to review with design.

From the updates, looks like we should review any use of native form elements, such as
* Radios
* Checkboxes
* Buttons
* Inputs and textareas

And regarding the global focus outline update, a few notes:
  * _The new focus indicator uses a thick dark ring with a thin white outline, which should improve visibility on both light and dark backgrounds_ - we should be sure to test the outline against both light and dark backgrounds
  * The new outline color is `#005FCC`. For reference, patternfly's `$pf-color-blue-400` (used as `$pf-global--primary-color--100`, `$pf-global--active-color--100`, and `$pf-global--link--Color`) is `#0066cc`, which is very similar, so the new focus outline color might make a component's styles look incorrect when the component receives focus.
  * The outline is more bold than it was previously, so we may also look specifically at elements that have an `outline-offset` declaration.

cc @mcarrano @jessiehuff 

--

@christiemolloy reported https://github.com/patternfly/patternfly/issues/3204

**Comment by mattnolting** [comment]
Focus states aren't appearing in FF because FF doesn't recognize dark background colors.

Here's an example:
![Screen Shot 2020-07-08 at 11 26 27 AM](https://user-images.githubusercontent.com/5385435/86938045-f6593b00-c10d-11ea-94d9-0fa2fb83c60e.png)
